### ISSUE TITLE
Linux mint - Unexpected error

### ISSUE THREAD

**Issue by MenthaSuaveolens** [bitwarden/desktop#1226]
### Steps To Reproduce

1. Start Bitwarden - Flatpack Version 1.27.1 Shell 11.4.5 Renderer 87.0.4280.141 Node 12.18.3 Architecture x64
2. Enter password
3. An unexpected error happened

Note that the Firefox-extension works perfectly
![Bitwarden-AnUnexpectedError](https://user-images.githubusercontent.com/47174957/149121581-5e1c3f5b-75d5-4b4b-88e2-56fc44199102.jpg)
 

### Expected Result

After properly entering the password that the vault opens

### Actual Result

Unexpected error

### Additional Context

Flatpak - installation from the standard LM software-repository

### Operating System

Linux

### Operating System Version

LM 20.2 xfce & Cinnamon, LM 20.3 xfce & Cinnamon

### Installation method

Other

### Build Version

Flatpack Version 1.27.1 Shell 11.4.5 Renderer 87.0.4280.141 Node 12.18.3 Architecture x64

**Comment by djsmith85** [comment]
Hi @MenthaSuaveolens could you please update to the latest flathub release 1.30.0 and see if that resolves the issue.

**Comment by MenthaSuaveolens** [comment]
I tried to install the new version, but it results in an empty dialog. flatpak install flathub com.bitwarden.desktop

Terminal output 
/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=bitwarden --file-forwarding com.bitwarden.desktop @@u %U @@
[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/Bitwarden/bitwarden --type=zygote
(node:2) DeprecationWarning: file property is deprecated and will be removed in v5.
(Use `bitwarden --trace-deprecation ...` to show where the warning was created)
Gtk-Message: 12:06:20.034: Failed to load module "xapp-gtk3-module"
[2:0112/120620.075084:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Bestand of map bestaat niet
libva error: /usr/lib/x86_64-linux-gnu/dri/intel-vaapi-driver/iHD_drv_video.so init failed
[43:0112/120620.186048:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[2:0112/120620.204937:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Bestand of map bestaat niet
[2:0112/120620.204996:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Bestand of map bestaat niet
[2:0112/120620.205022:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Bestand of map bestaat niet
LaunchProcess: failed to execvp:
xdg-settings
(node:2) electron: The default of nativeWindowOpen is deprecated and will be changing from false to true in Electron 15.  See https://github.com/electron/electron/issues/28511 for more information.
[43:0112/120620.294262:ERROR:shared_context_state.cc(73)] Skia shader compilation error
------------------------
#version 100

precision mediump float;
precision mediump sampler2D;
uniform sampler2D uTextureSampler_0_Stage0;
varying highp vec2 vlocalCoord_Stage0;
varying highp vec4 vtexSubset_Stage0;
void main() {
    mediump vec4 outputColor_Stage0 = vec4(1.0);
    highp vec2 texCoord;
    texCoord = vlocalCoord_Stage0;
    highp vec4 subset;
    subset = vtexSubset_Stage0;
    texCoord = clamp(texCoord, subset.xy, subset.zw);
    outputColor_Stage0 = texture2D(uTextureSampler_0_Stage0, texCoord);
    {
        gl_FragColor = outputColor_Stage0;
    }
}

Errors:
0:1(10): error: GLSL 4.40 is not supported. Supported versions are: 1.10, 1.20, 1.30, 1.40, 1.50, 3.30, 4.00, 4.10, 4.20, 1.00 ES, and 3.00 ES

**Comment by MenthaSuaveolens** [comment]
So I removed the latest flatpack. Removed also all data in $HOME/.var/app/com.bitwarden.desktop and the older version doesn't work any longer. Well, no big issue for me, because the firefox-extension works fine. Although there is an empty dialog, the menu is still responsive. Something gets wrong with the graphics.
![BitWarden](https://user-images.githubusercontent.com/47174957/149148539-e9528686-1dd6-4b0b-acdc-834b6871d90e.jpg)
